src/components/ChatApp/MessageSection/MessageSection.react.js

```diff
@@ -40,8 +40,8 @@ function getStateFromStores() {
     SnackbarOpenBackground: false,
     messages: MessageStore.getAllForCurrentThread(),
     thread: ThreadStore.getCurrent(),
-		currTheme: UserPreferencesStore.getTheme(),
-		tour:true,
```

Comment:
Next time make sure you don't commit the indentation changes made by your iDE. Make sure you commit only your changes.